test/Microsoft.AspNet.Mvc.FunctionalTests/PrecompilationTest.cs

```diff
@@ -149,17 +86,6 @@ private static string GetAssemblyNamePrefix()
             return typeof(Startup).GetTypeInfo().Assembly.GetName().Name + "." + nameof(RazorPreCompiler) + ".";
         }
 
-        private static async Task<string> TouchFile(string viewsDir, string file)
-        {
-            var path = Path.Combine(viewsDir, file);
-            File.AppendAllText(path, " ");
-
-            // Delay to allow the file system watcher to catch up.
-            await Task.Delay(_cacheDelayInterval);
-
-            return path;
-        }
-
         private sealed class ParsedResponse
```

Comment:
Well Roslyn shows the class has one reference but that's just because it has an explicit constructor :smiling_imp: 

Good catch; will remove.